[DOCS-6161] Update remap section and change VRL to DPL (#20787)
DataDog/documentation: content/en/observability_pipelines/reference/processing_language/_index.md

---
title: Datadog Processing Language
---

> The Datadog Processing Language (DPL) is an expression-oriented, domain-specific language designed for transforming data (logs and metrics (beta)) sent to Observability Pipelines. It features a simple syntax and [built-in functions][1].

Reviewer: ```suggestion
The Datadog Processing Language (DPL) is an expression-oriented, domain-specific language designed for transforming data (logs and metrics in beta) sent to Observability Pipelines. It features a simple syntax and [built-in functions][1].
```

Author: Going to leave this as is so that it's consistent with the other docs pages where `metrics (beta)` is used.

You can use DPL in the [`remap` transform][2] to:

- Manipulate [arrays][3], [strings][4], and other data types.
- Encode and decode values using [Codec][5].
- [Encrypt][6] and [decrypt][7] values.

> - [Coerce][8] one datatype to another datatype (for example, from an integer to a string).

Reviewer: ```suggestion
- [Coerce][8] one data type to another data type (for example, from an integer to a string).
```

> - [Convert syslog values][9] to read-able values.

Reviewer: ```suggestion
- [Convert syslog values][9] to readable values.
```
